### PR TITLE
Update ProgressRing Lottie File

### LINKED ISSUE #3787: Determinate ProgressRing not filling based off of the value entered
**Describe the bug**
There are a few issues with the determinate progress ring not filling the circle correctly when a particular value is set. These values are described below: 

Value of 50 isn't half of the circle:
image

 Values aren't determinate. Setting a value doesn't result in the same behavior all the time:
image

 The above means that in many cases there is no visual increment when incrementing between numbers (every value should show a discrete amount of the arc when the control is large enough on screen):
image

(I saw it go for 3-4 steps without moving, and then this seemed to cause a large jump between 99 and 100 at times.)

 The ring isn't completely filled at 100 (you can still see the end caps of the segment):
image

### LINKED ISSUE #5912: ProgressRing progress inconsistency
As shown in the xaml controls gallery, the ProgressRing does not fill consistently (with 'IsIndeterminate=False').

When the progress is set to 100 or 99, it shows like this:
image

Changing the value to 98 or 97 makes a huge difference:
image

Changing the value to 96 and below is consistent:
image
image
image

I would expect to have half a bar left at 50, but instead it's well over:
image

If this is expected behavior, then I would like to propose a 'IsAbsolute=true' property. I'm using the ProgressRing for a countdown timer and it looks very weird.

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

Update ProgressRing Determinate Lottie file so that its percentages matches the visuals.
There are now specific keyframes at 1%, 25%, 50%, 75%, 99%, 100%.

**1%**
image

**25%**
image

**50%**
image

**75%**
image

**99%**
image

**100%**
image

**Open question**: is this the desired visuals? 

## Motivation and Context

Fixes #3787 and fixes #5912

### CODE REVIEW

**Comment by harvinders** [comment]
@karenbtlai I don't think 99% is going to fly, it must show some gap.

**Comment by StephenLPeters** [comment]
Agreed, 99% is and should look noticeably different from 100%

File: dev/TabView/TestUI/TabViewPage.xaml [hunk 1]
```diff
@@ -135,7 +135,7 @@
 
                     <controls:TabView.Resources>
                         <ResourceDictionary>
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundSelected" Color="AliceBlue"/>
+                            <!--<SolidColorBrush x:Key="TabViewItemHeaderBackgroundSelected" Color="AliceBlue"/>-->
```
Comment (StephenLPeters) [inline dev/TabView/TestUI/TabViewPage.xaml:138]: Why is this getting commented out?

Comment (karenbtlai) [reply dev/TabView/TestUI/TabViewPage.xaml]: Oops, that was an accident. Reverted.

**Review by StephenLPeters — APPROVED** [review]

**Pull request merged**